setup.py

```diff
@@ -66,7 +66,7 @@
     url="https://github.com/ApeWorX/ape-hardhat",
     include_package_data=True,
     install_requires=[
-        "eth-ape==0.1.0a23",
+        "eth-ape==0.1.0a24",
```

Comment:
```suggestion
        "eth-ape>=0.1.0a24",
```

Hard-Pinning at this stage has turned out to be more of a pain than not I think (my bad!). Mostly because it is not a big deal if things break in alpha and it easier to upgrade to the latest ape without it.